styles.css

```diff
@@ -8,13 +8,13 @@ body,
 main,
 .container {
   width: 100%;
-  height: 100%
+  height: 100%;
 }
 
 main {
   display: flex;
   flex-direction: column;
-  height: 100%
+  /* height: 100%; */
```

Comment:
Lets delete  this comment out we are already deciding the width and height of html, body, main, and container set to the same value

Reply:
cool all set